jekyll/_cci2/workflows-waiting-status.md

```diff
@@ -19,3 +21,14 @@ Go to Settings > Branches in GitHub and click the Edit button on the protected b
 
 Refer to the [Orchestrating Workflows]({{ site.baseurl }}/2.0/workflows) document for examples and conceptual information.
 
+## GitHub Checks waiting for status in GitHub
+
+`ci/circleci:build — Waiting for status to be reported`
+
+If you have enabled GitHub Checks in your GitHub repository, but the status check never completes on Github Checks tab, there may be  status settings in GitHub that you need to deselect. For example, if you choose to protect your branches, you may need to deselect the `ci/circleci:build` status key as this check refers to the job status from CircleCI 2.0, as follows:
+
+![Uncheck GitHub Job Status Keys]({{ site.baseurl }}/assets/img/docs/github_job_status.png)
+
+Having the `ci/circleci:build` checkbox enabled will prevent the status from showing as completed in GitHub when using a GitHub Checks because CircleCI posts statuses to GitHub at a workflow level rather than a workflow job level.
```

Comment:
> because CircleCI posts statuses to GitHub at a workflow level rather than a workflow job level

Maybe this is the confusion that is the root cause of a lot of these. Is there any other place, early in the onboarding chain/docs that we can call this out, to cut them off at the pass?